### DISCUSSION TITLE
pooled connection string for postgres

### DISCUSSION THREAD

**Discussion by cborgia** [coollabsio/coolify#1520] [Feature Requests]
various cloud db services, like [neon.tech](https://neon.tech/) or [vercel postgres](https://vercel.com/docs/storage/vercel-postgres) or [planetscale](https://planetscale.com/) offer a way to show a db "connection string" as one line string, like coolify already does, or for various use uses e.g. nextjs, prisma, .env, django. it would be nice if coolify offered this. The question i have is not so much about connection string variations, but "pooled" (postgres)  or "optimized" strings 

Postgres cloud db services offer a "pooled" version of the connection string. 
Example from neon showing normal string, and pooled string:

**Normal** `postgresql://chris:************@ep-round-mountain-94973XXX.us-east-2.aws.neon.tech/neondb?sslmode=require`

**Pooled** `postgresql://chris:************@ep-round-mountain-94973XXX-pooler.us-east-2.aws.neon.tech/neondb?sslmode=require`

Mysql cloud db services offer a "direct" and "optimized" string:

**Direct** `DATABASE_URL='mysql://chris:************@us-east.connect.psdb.cloud/website?ssl={"rejectUnauthorized":true}'`
**Optimized** `DATABASE_URL='mysql://chris:************@aws.connect.psdb.cloud/website?ssl={"rejectUnauthorized":true}'`

Is this somethign that coolify can offer?

**Comment by AlejandroAkbal** [comment]
As a workaround, you could create a service like pgbouncer

Would love to see an automatic template for it too
